themes/vocabulary_theme/templates/blocks/recent-posts.html

```diff
@@ -16,15 +16,21 @@ <h2 class="is-paddingless level-left">
           {% set author = post.parent.parent.children.get('authors').children.get(post.author) %}
           <div class="column is-one-third is-paddingless padding-horizontal-big padding-top-bigger">
             <article class="card entry-post horizontal no-border blog-entry">
-              <header>
-                <figure class="image blog-image">
-                {% if author.about %}
-                  {% if author.md5_hashed_email %}
-                    <img class="profile" src="https://secure.gravatar.com/avatar/{{ author.md5_hashed_email }}?size=200"
-                    alt="gravatar" />
-                  {% endif %}
+              {% if author.about %}
+                {% if author.md5_hashed_email %}
+                  <header>
+                    <figure class="image blog-image">
+                      <img class="profile" src="https://secure.gravatar.com/avatar/{{ author.md5_hashed_email }}?size=200" alt="gravatar" />
+                    </figure>
+                  </header>
+                  {% else %}
+                  <header>
+                    <figure class="image blog-image">
+                      <img class="profile" src="./blank_profile.svg" alt="gravatar" />
```

Comment:
You can get the default Gravatar using the `d=` parameter. You can avoid the if/else using it.